docs/doc.md

```diff
@@ -36,23 +38,25 @@ class ButterFingersPerturbation(SentenceOperation):
     locales = ["en"]
 ```
 
-**Specificity:** While this is not a necessary criterion, it is highly encouraged to have a specific transformation. Eg. a perturbation which reverses the gender pronouns could give insights about gender bias in models, etc.
+**Specificity:** While this is not a necessary criterion, it is highly encouraged to have a specific transformation. Eg. a perturbation which only reverses the gender pronouns could give insights about gender bias in models, etc.
 
-**Adding New Libraries:** We welcome addition of libraries which are light and can be installed via pip. Every library should specify the version number associated and be added in requirements.txt. 
+**Adding New Libraries:** We welcome addition of libraries which are light and can be installed via pip. Every library should specify the version number associated and be added in requirements.txt. However, it's not highly encouraged to use heavy libraries for a few lines of code which can be replicated directly.
 
 **Description:** The `README.md` file should clearly explain what the transformation is attempting to generate as well as the importance of that transformation for the specified tasks.
 
-**Paraphrasers and Data Augmenters:** Besides perturbations, we strongly encourage transformation methods which act like paraphrasers and data augmenters. For non-deterministic approaches, we encourage you to specify metrics which could provide an estimate of the quality of generation. It is preferred to have a high precision transformation generator as compared to a low accuracy one. And hence it's okay to selectively generate.
+**Paraphrasers and Data Augmenters:** Besides perturbations, we strongly encourage transformation methods which act like paraphrasers and data augmenters. For non-deterministic approaches, we encourage you to specify metrics which could provide an estimate of the quality of generation. It is preferred to have a high precision transformation generator as compared to a low accuracy one. And hence it's okay if your transformation selectively generates. If your transformation loads a deep-learning model, especially a heavy one (like BERT or T5 or their cousins), set the `heavy` variable to `True`.
```

Comment:
Besides perturbations, we ~strongly encourage~ welcome transformation methods ~which~ that act like paraphrasers and data augmenters. For non-deterministic approaches, we encourage you to specify metrics which ~could~can provide an estimate of the ~quality of~ generation quality. It is preferred to have a high precision transformation generator ~as~ compared to a low accuracy one. And hence it's okay if your transformation selectively generates. If your transformation loads a deep-learning model, especially a heavy one (like BERT or T5 or their cousins), set the `heavy` variable to `True`.